llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -764,11 +764,22 @@ define void @foo(ptr %arg0, ptr %arg1) {
   // Check create(InsertBefore)
   sandboxir::LoadInst *NewLd =
       sandboxir::LoadInst::create(Ld->getType(), Arg1, Align(8),
-                                  /*InsertBefore=*/Ret, Ctx, "NewLd");
+                                  /*InsertBefore=*/Ret, Ctx, 
+                                  Ld->isVolatile(), "NewLd");
```

Comment:
Replace `Ld->isVolatile()` with `/*IsVolatile=*/false`

Reply:
Fixed it